app/models/user.rb

```diff
@@ -33,4 +33,17 @@ def followers
     end
     list
   end
+
+  def following
+    conn = Faraday.new(url: 'https://api.github.com') do |faraday|
+      faraday.headers['Authorization'] = "token #{token}"
+    end
```

Comment:
We can probably make this a separate method since it has been repeated multiply times in our github section